### ISSUE TITLE
How to position window in top left corner of screen?

### ISSUE THREAD

**Issue by planet36** [sineemore/stw#3]
The description of px, py in config.h is unclear:
> px, py values position window top left corner in the center of the screen

The usage message doesn't describe the difference between -x, -X, and -y, -Y.
What are the units of value? Pixels, percentage?

I've tried combinations of 0%, -50% for -x, -y, -X, and -Y, but the window is never positioned in the upper left corner of the screen?
The command being run is `pstree -U`.

**Comment by sineemore** [comment]
```
stw -x0 -y0 pstree -U
```

The description is really misleading.

-x and -y params position one of the window corners determined by signs in values. Positive sign (or a plain 0) stands for top and left sides, negative for bottom and right. For example `-x -0 -y -0` will position bottom right window corner in the bottom right screen corner, `-x 0 -y -0` will do the same for bottom left.

Percentage values in -x and -y params refer to screen width and height. For example `-x 50% -y 50%` will position top left window corner in center of the screen.

-X and -Y (capitals) is a second pair of params to position window. Values from this params are added directly to values in -x and -y. Also percentage values refer the window size itself.

So, to sum up, if we want a centered stw window will go with:
* `-x 50% -y 50%` to move the top left corner in the center of the screen
* and `-X -50% -Y -50%` to move the window up and to the left by height * 50% and width * 50% units

```
stw -x 50% -y 50% -X -50% -Y -50% /bin/echo "I am centered"
```

**Comment by sineemore** [comment]
Also note, with `stw -x 0 -y 0` the window will be literally in the top left corner of the screen, probably behind a bar if you have some. Adjust the values in -y, for example `-x 0 -y 32`.

**Comment by planet36** [comment]
![screenshot](https://user-images.githubusercontent.com/387896/96357740-a9521100-10cd-11eb-8042-f26f7fb05897.png)

**Comment by planet36** [comment]
I got the desired result by changing config.h

![screenshot](https://user-images.githubusercontent.com/387896/96358166-149de200-10d2-11eb-918f-3efeb6bf77e6.png)

**Comment by sineemore** [comment]
Oh, I got it. I've added bad default values to config.h. Will revert tx, ty variables back to `{0}`.

**Comment by planet36** [comment]
It works now.
Thanks.
